collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
 ---
 layout:        infographic
-title:         "What is Fit for 55"
+title:         "What is 'Fit for 55'"
```

Comment:
definitely add a ? "'What is Fit for 55'?" or do a direct translation "Fit for 55: overview of suggested measures"

Reply:
Thanks. Updated.